sqlglot/dialects/clickhouse.py

```diff
@@ -21,6 +21,16 @@ def _lower_func(sql: str) -> str:
     return sql[:index].lower() + sql[index:]
 
 
+def _quantile(self, e):
+    quantile = e.args.get("quantile")
```

Comment:
We can omit the `get` call here since `quantile` is a required arg:

```suggestion
    quantile = e.args["quantile"]
```